src/components/PhotoPairGame.tsx

```
@@ -38,22 +24,54 @@ const heartLayout: CellType[][] = [
   [null, null, null, null, null, null, null],
 ];
 
-type ValentinesProposalProps = {
+type PhotoPairGameProps = {
+  images?: string[]; // expects 8 images; falls back to defaultPairs
   handleShowProposal: () => void;
 };
 
+const shuffleArray = (array: string[]) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'shuffleArray'. Consider to delete it or rename it.

'shuffleArray' is defined here:


(lint/suspicious/noRedeclare)